promql/printer_test.go

```diff
@@ -94,7 +94,10 @@ func TestExprString(t *testing.T) {
 			in: `a - ON(b) GROUP_LEFT(x, y) c`,
 		},
 		{
-			in: `a - ON(b) GROUP_LEFT c`,
```

Comment:
I wouldn't change the input, but make the new behavior clear by adding the `out` attribute.